manager/controllers/app/select_modules.go

```diff
@@ -40,7 +40,7 @@ func (p *PathBuilder) solve() (Solution, error) {
 		p.Log.Error().Str(logging.DATASETID, p.Asset.Context.DataSetID).Msg(msg)
```

Comment:
These log printings can be done inside some util function shared by this implementation and the solver.